templates/base.html.twig

```diff
@@ -5,7 +5,6 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <title>{% block title %}{% endblock %} - Community Event Manager</title>
         <link rel="stylesheet" href="{{ asset('build/base.css') }}" />
```

Comment:
Can you use `encore_entry_link_tags` instead of `asset`

Also if you can move these in the `stylesheets` block

Reply:
I'm blocking to set up even_entry_link_tags. Obviously I have to remove webpack_encore_pack to put webpack_encore_bundle. And despite that I have a symfony arrery that does not find "entrypoint.json"